test/hotspot/jtreg/runtime/cds/appcds/TestZGCWithCDS.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Here and later: for the files with no changes, no need to update this line, I think.

Reply:
I reverted the files with only copyright year changes.